acceptance/tests/provider/package/yum_semantic_versioning.rb

```diff
@@ -0,0 +1,58 @@
+test_name "yum provider should use semantic versioning for ensuring desired version" do
+  confine :to, :platform => /el7/
```

Comment:
I don't think there is any platform that contains `/el7/` so this test will not run on anything. I checked in jenkins and it did not run on redhat-7


```
10:59:45 Begin tests/provider/package/yum_semantic_versioning.rb
10:59:45 
10:59:45 yum provider should use semantic versioning for ensuring desired version
10:59:45   Warning: No suitable hosts with: {:platform=>/el7/}
10:59:45   
10:59:45   No suitable hosts found with {:platform=>/el7/}
10:59:45 Begin teardown
10:59:45 End teardown
10:59:45 Warning: tests/provider/package/yum_semantic_versioning.rb skipped in 0.00 seconds
```

You can check the platform using beaker-hostgenerator

```
❯ beaker-hostgenerator redhat7-64
---
HOSTS:
  redhat7-64-1:
    pe_dir:
    pe_ver:
    pe_upgrade_dir:
    pe_upgrade_ver:
    platform: el-7-x86_64
    packaging_platform: el-7-x86_64
    template: redhat-7-x86_64
    hypervisor: vmpooler
    roles:
    - agent
```